internal/hud/renderer.go

```diff
@@ -340,24 +314,49 @@ func (r *Renderer) renderResource(res view.Resource, rv view.ResourceViewState,
 			}
 		}
 	}
-
 	if len(buildComponents) == 0 {
 		buildComponents = []rty.Component{rty.TextString("no build yet")}
 	}
-
 	l := rty.NewLine()
 	l.Add(rty.ColoredString("  BUILD: ", cLightText))
 	l.Add(buildComponents[0])
 	layout.Add(l)
-
 	for _, c := range buildComponents[1:] {
 		l := rty.NewLine()
 		l.Add(rty.TextString("         "))
 		l.Add(c)
 		layout.Add(l)
 	}
+	layout.Add(rty.NewLine())
+}
+
+func renderResourceSummary(selected bool, rv view.ResourceViewState, res view.Resource, layout *rty.ConcatLayout) {
+	l := rty.NewLine()
+	sb := rty.NewStringBuilder()
+	p := "  "
+	if selected {
+		p = "▼ "
+	}
+	if selected && rv.IsCollapsed {
+		p = "▶ "
+	}
 
-	// Kubernetes Info ---------------------------------------
+	sb.Text(p)
+	sb.Textf("%s ", res.Name)
+
+	l.Add(sb.Build())
+	l.Add(rty.NewFillerString('╌'))
```

Comment:
this is bikesheddy and I'm not sure how to have a constructive conversation about this, but curious if you have more concrete thoughts than I do on broken lines vs continuous lines